template/template.go

```diff
@@ -276,6 +279,13 @@ func NewTemplateExpander(
 				}
 				return float64(time.Duration(v)) / float64(time.Second), nil
 			},
+			"parseGoDuration": func(d string) (time.Duration, error) {
+				v, err := model.ParseDuration(d)
+				if err != nil {
+					return 0, err
+				}
+				return time.Duration(v), nil
+			},
```

Comment:
To stay more in line with the other functions (i.e. deal with the Prometheus types everywhere but offer a conversion functions for Go types), let's simply enable negative durations above in `parseDuration` and instead of `parseGoDuration`, offer a `toDuration` functions similar to the existing `toTime` function.